src/TLuaInterpreter.cpp

```diff
@@ -1357,7 +1357,7 @@ int TLuaInterpreter::paste(lua_State* L)
     return 0;
 }
 
-
+// The text MUST be presented in the ENCODING selected to be used by the SERVER
```

Comment:
I wasn't writing the Wiki at the time and it caught me out - it does need to be mentioned there **as well**...

Reply:
It's just not a useful comment to have here. We can sprinkle the whole codebase with such random comments - it describes nothing about what the function does, none of the inputs, none of the outputs, just picks on a selected facet of its behaviour. Or we could have one place where it's all consistently documented :thinking:

Reply:
Well, in that case I guess I will have to extend the comment to include those details, um:
"Takes one argument, a string and sends it to the trigger processing system almost as if it came from the MUD Server.  This string must be byte encoded in a manner to match the currently selected Server Encoding.  The trigger processing system will recognise that this data is internal and, should it be retaining a few bytes from the MUD server because a previous character was split between two network packets, will not try to 'cross the streams'<sup>1 (:film_projector: we do not want to do that)</sup> and prepend those stored bytes - instead it will hang on to them until the next network packet is processed.  There are no values returned to the lua caller and an lua error is thrown if the supplied argument is not a lua string."
     
     
<sub>**Footnote:**
Egon: Don’t cross the streams.
Peter: Why?
Egon: It would be bad.
Peter: I’m fuzzy on the whole good/bad thing. What do you mean “bad”?
Egon: Try to imagine all life as you know it stopping instantaneously and every molecule in your body exploding at the speed of light.
Raymond: Total protonic reversal.
Peter: That’s bad. Okay. Alright, important safety tip, thanks Egon.
<right>-- Ghostbusters (1984)</right></sub>

Reply:
Yeah, and all that could be on the wiki, since that's what mostly what the wiki says about it already!

Reply:
📖  Have updated the Wiki to also mention the need to encode the text into the same form as the Mud Server Encoding setting.